### PR TITLE
[FR] Mapping techniques stubs

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the following wiki pages:

- Mapping_Techniques/Blanket
- Mapping_Techniques/Overlap
- Mapping_Techniques/Stack
- Mapping_Techniques/Time-distance_equality

**Notify me if the PR is merge without review, that I self-review before.**

### CODE REVIEW

File: wiki/Mapping_Techniques/Blanket/fr.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+tags:
+  - wrapping
+  - blanketing
+---
+
+# Blanket
+
+Un **blanket** est l'effet créé lorsqu'un [sliderbody](/wiki/Hit_object/Sliderbody) s'enroule autour de la forme d'un autre [objet](/wiki/Hit_object). le blanket peut être réalisée de plusieurs façons pour s'adapter au style visuel d'une [beatmap](/wiki/Beatmap). Vous trouverez un guide pour les réaliser dans l'article sur les [techniques de mapping](/wiki/Mapping_Techniques/Making_Good_Sliders#beat-blankets).
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Blanket/fr.md:10]:
```suggestion
Un **blanket** est l'effet créé lorsqu'un [sliderbody](/wiki/Hit_object/Sliderbody) s'enroule autour de la forme d'un autre [objet](/wiki/Hit_object). Le blanket peut être réalisé de plusieurs façons pour s'adapter au style visuel d'une [beatmap](/wiki/Beatmap). Vous trouverez un guide pour les réaliser dans l'article sur les [techniques de mapping](/wiki/Mapping_Techniques/Making_Good_Sliders#beat-blankets).
```

File: wiki/Mapping_Techniques/Overlap/fr.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overlap
+
+![](img/overlap.jpg "Exemple d'un hit circle recouvrant un sliderhead.")
+
+Un **overlap** apparaît lorsque des [objets](/wiki/Hit_Objects) se touchent sans créer de [stacking](/wiki/Mapping_Techniques/Stack).
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Overlap/fr.md:9]: "chevauchent" plutôt que "touchent" me semble plus clair?

File: wiki/Mapping_Techniques/Stack/fr.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Stack
+
+![](img/stack.jpg "Un exemple de deux cercle empilés sur un sliderhead.")
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Stack/fr.md:7]:
```suggestion
![](img/stack.jpg "Un exemple de deux cercles empilés sur un sliderhead.")
```

File: wiki/Mapping_Techniques/Stack/fr.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Un **stack** est un ensemble d'[objets](/wiki/Hit_Objects) qui sont en [overlap](/wiki/Mapping_Techniques/Overlap) les uns sur les autres dans le champ de jeu. Les objets les plus courants qui sont empilés sont les [cercles](/wiki/Hit_Objects#hit-circle).
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Stack/fr.md:9]:
```suggestion
Un **stack** est un ensemble d'[objets](/wiki/Hit_Objects) qui sont en [overlap](/wiki/Mapping_Techniques/Overlap) les uns sur les autres dans le champ de jeu. Les objets les plus courants qui sont stackés sont les [cercles](/wiki/Hit_Objects#hit-circle).
```

File: wiki/Mapping_Techniques/Stack/fr.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Les stacks sont automatiquement créées lorsque les objets sont proches de se chevaucher parfaitement. Le temps entre les objets pour que cela se produise est déterminé par l'indulgence du stacking de la [beatmap](/wiki/Beatmap).
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Stack/fr.md:11]:
```suggestion
Les stacks sont automatiquement créés lorsque les objets sont proches de se chevaucher parfaitement. Le temps entre les objets pour que cela se produise est déterminé par l'indulgence du stacking de la [beatmap](/wiki/Beatmap).
```

File: wiki/Mapping_Techniques/Time-distance_equality/fr.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+tags:
+  - distance snap
+  - DS
+---
+
+# Time-distance equality
+
+Le **time-distance equality** est une [technique de mapping](/wiki/Mapping_Techniques) du mode [osu!](/wiki/Game_mode/osu!) où l'espacement entre les [objets](/wiki/Hit_object) sur le terrain de jeu est proportionnel au temps qui les sépare. Cet espacement est automatiquement calculé par la fonction [distance snap](/wiki/Beatmap_Editor/Distance_Snap) de l'[éditeur de beatmap](/wiki/Beatmap_Editor).
+
+Dans les [difficultés](/wiki/Beatmap/Difficulty) inférieures, le time-distance equality est utilisée pour rendre le jeu plus intuitif pour les nouveaux joueurs.
```
Comment (osuShiro) [inline wiki/Mapping_Techniques/Time-distance_equality/fr.md:12]:
```suggestion
Dans les [difficultés](/wiki/Beatmap/Difficulty) inférieures, le time-distance equality est utilisé pour rendre le jeu plus intuitif pour les nouveaux joueurs.
```

**Review by osuShiro — CHANGES_REQUESTED** [review]

File: wiki/Mapping_Techniques/Overlap/fr.md (hunk 2 above)
Comment (DoryanR) [reply wiki/Mapping_Techniques/Overlap/fr.md]: Oui, ce sera plus clair a mon avis

**Review by osuShiro — APPROVED** [review]

**Pull request merged**